pkg/flare/archive_test.go

```diff
@@ -506,3 +509,75 @@ func TestZipVersionHistory(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "mockfilecontent", string(actualContent))
 }
+
+func TestZipProcessAgentFullConfig(t *testing.T) {
+	type ProcessConfig struct {
+		Enabled string `yaml:"enabled"`
+	}
+
+	globalCfg := struct {
+		Apikey     string        `yaml:"api_key"`
+		DDurl      string        `yaml:"dd_url"`
+		ProcessCfg ProcessConfig `yaml:"process_config"`
+	}{
+		"*****1234",
```

Comment:
I suspect you wanted this un-scrubbed?

Reply:
I've ended up using `*****1234` because process-agent already returns the configs scrubbed. However, I've updated the test to make sure that the core agent is also obfuscating it if it's necessary.